WooCommerce/Classes/ViewRelated/Products/Add Product/AddProductWithAI/Survey/ProductCreationAISurveyConfirmationViewModel.swift

```diff
@@ -0,0 +1,26 @@
+import UIKit
+
+/// View model for `ProductCreationAISurveyConfirmationView`.
+struct ProductCreationAISurveyConfirmationViewModel {
+    private let onTappingStartTheSurvey: () -> Void
+    private let onTappingSkip: () -> Void
+    private let analytics: Analytics
+
+    init(onTappingStartTheSurvey: @escaping () -> Void,
+         onTappingSkip: @escaping () -> Void,
```

Comment:
Nit: can we simplify the naming to reduce verbosity? 

```suggestion
    init(onStart: @escaping () -> Void,
         onSkip: @escaping () -> Void,
```

Reply:
Done in https://github.com/woocommerce/woocommerce-ios/pull/11390/commits/a00c790a7d39be347df7c473dd34509fc3a9205f